javascript/javascript2/week1/hyf-bay/main.js

```diff
@@ -0,0 +1,54 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+
+//function get color is to get a random color
+function getColor() {
+  let rgb1 = Math.floor(Math.random() * 256) + 1;
+  let rgb2 = Math.floor(Math.random() * 256) + 1;
+  let rgb3 = Math.floor(Math.random() * 256) + 1;
+  return "color : rgb(" + rgb1 + "," + rgb2 + "," + rgb3 + ")";
+}
+
+function getRating(ratingValue) {
+  let ratingStar = "";
+  for (let i = 0; i < ratingValue; i++) {
+    ratingStar += "*";
+  }
+  return ratingValue + " : " + ratingStar;
+}
```

Comment:
In this case, rating should be in number count, instead of stars. Because if you want to show it in stars then it should be percentage out of 5 stars. In this case you are just adding the count so its better to show it in numbers.

Reply:
It shows both number count and stars.